xla/service/gpu/amdgpu_compiler.cc

```diff
@@ -178,16 +179,19 @@ absl::Status AMDGPUCompiler::OptimizeHloPostLayoutAssignment(
     HloModule* hlo_module, se::StreamExecutor* stream_exec,
     const CompileOptions& options, const TargetConfig& gpu_target_config,
     tsl::thread::ThreadPool* thread_pool) {
-  HloPassPipeline pre_pipeline("AMDGPU post-layout_assignment part 1");
-
-  auto rocm_compute_capability = std::get<se::RocmComputeCapability>(
+  auto rocm_cc = std::get<se::RocmComputeCapability>(
       gpu_target_config.device_description.gpu_compute_capability());
 
+  HloPassPipeline pre_pipeline("AMDGPU post-layout_assignment part 1");
+  bool allow_hipblaslt = rocm_cc.gfx_version() == "gfx950";
```

Comment:
maybe add `has_mxdata_support()` here via `rocm_cc` https://github.com/ROCm/xla/blob/rocm-jaxlib-v0.6.0/xla/stream_executor/device_description.h#L100 ? so you wouldn't add one gfx hard coding here